testnets/pocket/chain.json

```diff
@@ -0,0 +1,119 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "pocket-beta",
+  "status": "live",
+  "network_type": "testnet",
+  "website": "https://www.pokt.network/",
+  "pretty_name": "Pocket Beta TestNet",
+  "chain_type": "cosmos",
+  "chain_id": "pocket-beta",
+  "bech32_prefix": "pokt@",
+  "daemon_name": "poktrolld",
+  "node_home": "$HOME/.poktroll",
+  "key_algos": ["secp256k1"],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "upokt",
+        "fixed_min_gas_price": 0.001,
+        "low_gas_price": 0.001,
+        "average_gas_price": 0.002,
+        "high_gas_price": 0.0025
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "upokt"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/pokt-network/poktroll",
+    "recommended_version": "v0.0.11-rc",
+    "compatible_versions": ["v0.0.11-rc"],
+    "binaries": {
+      "linux/amd64": "https://github.com/pokt-network/poktroll/releases/download/v0.0.11-rc/poktroll_linux_amd64.tar.gz",
+      "linux/arm64": "https://github.com/pokt-network/poktroll/releases/download/v0.0.11-rc/poktroll_linux_arm64.tar.gz",
+      "darwin/amd64": "https://github.com/pokt-network/poktroll/releases/download/v0.0.11-rc/poktroll_darwin_amd64.tar.gz",
+      "darwin/arm64": "https://github.com/pokt-network/poktroll/releases/download/v0.0.11-rc/poktroll_darwin_arm64.tar.gz"
+    },
+    "consensus": {
+      "type": "cometbft",
+      "version": "v0.38.10"
+    },
+    "genesis": {
+      "name": "v0.0.11-rc",
+      "genesis_url": "https://github.com/pokt-network/pocket-network-genesis/blob/master/shannon/testnet/genesis.json"
+    },
+    "versions": [
+      {
+        "name": "v0.0.11-rc",
+        "tag": "v0.0.11-rc",
+        "height": 1,
+        "next_version_name": "v0.0.12"
+      }
+    ],
+    "sdk": {
+      "type": "cosmos",
+      "version": "v0.50.9"
+    },
+    "ibc": {
+      "type": "go",
+      "ics_enabled": []
+    },
+    "cosmwasm": {
+      "enabled": false
+    }
+  },
+  "images": [
+    {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/pocket/images/pokt-icon.png",
+      "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/pocket/images/pokt-icon.svg"
```

Comment:
the png and svg cannot be in the same image object because they are visually different (one has white fill, which the other is transparent inside). Same issue for the logo_URIs property. Either use the same image for svg and png, or place the svg and png into different objects.